packages/server/graphql/public/mutations/addFeatureFlagToOrg.ts

```diff
@@ -0,0 +1,34 @@
+import getRethink from '../../../database/rethinkDriver'
+import {getUserId, isSuperUser} from '../../../utils/authorization'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const addFeatureFlagToOrg: MutationResolvers['addFeatureFlagToOrg'] = async (
+  _source,
+  {orgIds, flag},
+  {authToken}
+) => {
+  const r = await getRethink()
+
+  // AUTH
+  const viewerId = getUserId(authToken)
+
+  if (!isSuperUser(authToken)) {
+    return standardError(new Error('Not authorised to add feature flag'), {
+      userId: viewerId
+    })
+  }
+
+  // RESOLUTION
+  await r
+    .table('Organization')
+    .getAll(r.args(orgIds), {index: 'id'})
```

Comment:
+1 we can just do `.getAll(orgIds)` here. Here's the ChatGPT explanation 😉 :
![chatgpt-query](https://user-images.githubusercontent.com/39854876/206224991-ea20833e-76ba-44e5-b446-9c8350c8b1c3.png)